Bottom: 3b981f460974a986e47808ca71f3d39fe078b5cc
Top:    023cfb0b233cc6fc70f8d251d2b6c163e7b36627
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-04-08 04:16:27 -0300

Refresh of access-control.patch

---

diff --git a/libcc1/libcp1plugin.cc b/libcc1/libcp1plugin.cc
index d91f393..f896afc 100644
--- a/libcc1/libcp1plugin.cc
+++ b/libcc1/libcp1plugin.cc
@@ -954,7 +954,7 @@ plugin_new_friend (cc1_plugin::connection * /* self */,
   /* FIXME: is this enough to support template friend declarations?  */
 
   if (TYPE_P (decl))
-    make_friend_type (current_class_type, decl, true);
+    make_friend_class (current_class_type, decl, true);
   else
     {
       DECL_FRIEND_P (decl) = true;
